### ISSUE TITLE
Error detecting vulnerability

### ISSUE THREAD

**Issue by ghost** [vanhoefm/krackattacks-scripts#28]
./krack-test-client.py
Traceback (most recent call last):
  File "./krack-test-client.py", line 12, in <module>
    from libwifi import *
  File "/home/nicolas/krackattacks-scripts/krackattack/libwifi.py", line 6, in <module>
    from Cryptodome.Cipher import AES
ImportError: No module named Cryptodome.Cipher

**Comment by jmyth742** [comment]
have you installed all the correct packages for this? double check as the 'python-pycryptodome'  sometimes doesnt install (outside kali distro)

**Comment by Burrch3s** [comment]
I ran into that my first try on Ubuntu(technically XUbuntu) but the steps at the end of the readme are the same troubleshooting steps I tried on my own and it worked for me. I believe I may have had to run the, `pip install pycryptodomex` command to get the correct modules. If that still doesn't work for you check out "http://pycryptodome.readthedocs.io/en/latest/src/installation.html" to install, but the above command worked for me so I don't imagine it will give you too much more trouble.
